internal/pkg/deploy/cloudformation/stack/pipeline_test.go

```diff
@@ -99,16 +99,16 @@ func mockCreatePipelineInput() *deploy.CreatePipelineInput {
 		},
 		ArtifactBuckets: []deploy.ArtifactBucket{
 			{
-				BucketArn: "arn:aws:s3:::chicken-us-east-1",
```

Comment:
We actually do need the ARN. It's also more consistent that way because for now the stack instance created via the stackset returns the KMS key ARN and bucket name. I'm sending out a PR that changes the stackset to return bucket ARN instead of bucket name.

Reply:
Ping @kohidave